contracts-periphery/test/DeployBatchSend.t.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import {Deploy} from "script/DeployBatchSend.s.sol";
+import {Test} from "forge-std/Test.sol";
+import {console2} from "forge-std/console2.sol";
+
+contract DeployBatchSendTest is Deploy, Test {
+  uint256 constant EXPECTED_TEST_NONCE = 4; // TODO Edit this with the nonce of the
+  address expectedContractAddress = computeCreateAddress(msg.sender, EXPECTED_NONCE);
+
+  function setUp() public {
+    assertEq(EXPECTED_NONCE, EXPECTED_TEST_NONCE);
+    assertTrue(expectedContractAddress.code.length == 0);
+  }
+
+  function test_checkContractIsDeployed() public {
+    console2.log("expectedContractAddress is ", expectedContractAddress);
+    run();
+    for (uint256 i; i < networks.length; i++) {
+      vm.createSelectFork(getChain(networks[i]).rpcUrl);
```

Comment:
didn't we already create the forks in the `run` method? here we should just be selecting an already existing fork, not creating a new one

Reply:
Latest test in https://github.com/ScopeLift/umbra-protocol/pull/504/commits/964caa73ebe402ebe5f5e0640cb87f7f5137c9fc